### PR TITLE
docs(self-managed): add Identity contextPath in sm troubleshooting

### PR DESCRIPTION
**Pull request by aabouzaid**
## Description

I've seen many users stumbled into this, so I added Identity contextPath limitation where it works in HTTPS only.

## When should this change go live?

ASAP.

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

I will move the change to v8.1 and v8.2 after the review.

- [x] I have added changes to the relevant `/versioned_docs` directory.
- [x] I have added changes to the main `/docs` directory (aka `/next/`).
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

### CODE REVIEW

**Comment by aabouzaid** [comment]
@Ben-Sheppard could you please take a look before the technical review? :pray:

File: versioned_docs/version-8.2/self-managed/platform-deployment/troubleshooting.md [hunk 1]
```diff
@@ -43,4 +43,4 @@ However, according to the official Kubernetes documentation about [Ingress TLS](
 
 > There is a gap between TLS features supported by various Ingress controllers. Please refer to documentation on nginx, GCE, or any other platform specific Ingress controller to understand how TLS works in your environment.
 
-Therefore, if you are not using the [Ingress-NGINX controller](https://github.com/kubernetes/ingress-nginx), ensure you pay attention to TLS configuration of the Ingress controller of your choice. Find more details about the Zeebe Ingress setup in the [Kubernetes platforms supported by Camunda](./helm-kubernetes/platforms/platforms.md).
+Therefore, if you are not using the [Ingress-NGINX controller](https://github.com/kubernetes/ingress-nginx), ensure you pay attention to TLS configuration of the Ingress controller of your choice. Find more details about the Zeebe Ingress setup in the [Kubernetes platforms supported by Camunda](./helm-kubernetes/platforms/platforms.md). Please note also that the contextPath doesn't work in browser Incognito mode.
```
Comment (Ben-Sheppard) [inline versioned_docs/version-8.2/self-managed/platform-deployment/troubleshooting.md:46]: I think here I would use a slightly more positive tone, IMO `doesn't work` implies that something is broken or neglected whereas the limitation here is imposed on us by the technologies in use. 

```suggestion
Therefore, if you are not using the [Ingress-NGINX controller](https://github.com/kubernetes/ingress-nginx), ensure you pay attention to TLS configuration of the Ingress controller of your choice. Find more details about the Zeebe Ingress setup in the [Kubernetes platforms supported by Camunda](./helm-kubernetes/platforms/platforms.md). 

:::note
Due to limitations, the contextPath approach is unavailable when using a browser Incognito mode.
:::
```

File: docs/self-managed/platform-deployment/troubleshooting.md [hunk 2]
```diff
@@ -44,3 +44,9 @@ However, according to the official Kubernetes documentation about [Ingress TLS](
 > There is a gap between TLS features supported by various Ingress controllers. Please refer to documentation on nginx, GCE, or any other platform specific Ingress controller to understand how TLS works in your environment.
 
 Therefore, if you are not using the [Ingress-NGINX controller](https://github.com/kubernetes/ingress-nginx), ensure you pay attention to TLS configuration of the Ingress controller of your choice. Find more details about the Zeebe Ingress setup in the [Kubernetes platforms supported by Camunda](./helm-kubernetes/platforms/platforms.md).
+
+## Identity contextPath
+
+Camunda Platform 8 Self-Managed could be accessed externally via different method, one of them is the [combined Ingress setup](./helm-kubernetes/guides/ingress-setup.md#combined-ingress-setup). In that model, Camunda Identity is accessed by the contextPath, for example `https://camunda.example.com/identity`.
```
Comment (Ben-Sheppard) [inline docs/self-managed/platform-deployment/troubleshooting.md:50]:
```suggestion
Camunda Platform 8 Self-Managed can be accessed externally via different methods, one such method is the [combined Ingress setup](./helm-kubernetes/guides/ingress-setup.md#combined-ingress-setup). In that configuration, Camunda Identity is accessed using a specific path, configured by setting the contextPath variable, for example `https://camunda.example.com/identity`.
```

File: docs/self-managed/platform-deployment/troubleshooting.md [hunk 3]
```diff
@@ -44,3 +44,9 @@ However, according to the official Kubernetes documentation about [Ingress TLS](
...
+
+However, for security reasons, Camunda Identity can be accessed via contextPath in secure mode only (HTTPS). If you want to use Camunda Identity with HTTP, then use [separate Ingress setup](./helm-kubernetes/guides/ingress-setup.md#combined-ingress-setup) (you still can use the rest of the applications like Operate, Optimize, etc, in combined setup).
```
Comment (Ben-Sheppard) [inline docs/self-managed/platform-deployment/troubleshooting.md:52]:
```suggestion
For security reasons, Camunda Identity requires secure access (HTTPS) when a `contextPath` is configured. If you want to use Camunda Identity with HTTP, please use a [separate Ingress setup](./helm-kubernetes/guides/ingress-setup.md#combined-ingress-setup) (applications such as Operate, Optimize, etc, can still be accessed in a combined setup).
```

Comment (Ben-Sheppard) [reply docs/self-managed/platform-deployment/troubleshooting.md:52]: Also, this link points to the `combined-ingress-setup` heading, is that correct?

**Review by Ben-Sheppard — CHANGES_REQUESTED** [review]
A couple of suggestions :)

File: docs/self-managed/platform-deployment/troubleshooting.md (hunk 3 above)
Comment (aabouzaid) [reply docs/self-managed/platform-deployment/troubleshooting.md:52]: good catch, I will fix the heading :ok_hand:

**Comment by aabouzaid** [comment]
Thanks @Ben-Sheppard for suggestions :ok_hand: 
And added @akeller for tech review since @christinaausley is off.

**Review by Ben-Sheppard — APPROVED** [review]
From my perspective this looks good! :)

**Review by akeller — COMMENTED** [review]
contextPath is a variable, so let's wrap it in backticks.

I'm doing quick reviews this morning and can come back to this PR and make the changes myself (unless you get to it first 🙂 ).

**Comment by akeller** [comment]
Back to make the fixes so we can merge this, give me a few min ⌨️

**Pull request merged**